package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "clean": "rimraf lib",
+    "clean:game": "ts-node -e 'const { databaseUtil } = require(\"./src/utils/databaseUtil.ts\"); databaseUtil.delete();'",
```

Comment:
`ts-node` isn't doing anything differently here than `node`. I'd revert it back to `node`. I have a suggestion that would be easier to show in a separate PR

Reply:
Actually why not just do `rm -r saves/`?

Reply:
> `ts-node` isn't doing anything differently here than `node`. I'd revert it back to `node`. I have a suggestion that would be easier to show in a follow up PR

Are you saying i should revert to `"clean:game": "node -e 'const { databaseUtil } = require(\"./src/utils/databaseUtil.ts\"); databaseUtil.delete();'",` or are you saying to revert to `"clean:game": "node -e 'const { databaseUtil } = require(\"./lib/utils/databaseUtil.js\"); databaseUtil.delete();'",`?

The first one will cause `SyntaxError: Cannot use import statement outside a module` and the second one can only be used if you run `npm run prepack`.

> Actually why not just do `rm -r saves/`?

The first version of this command was that but then I look at the task and it stated to call `delete`. I'm fine with whichever.

Reply:
I like `rm -r saves`. It's simple.

Reply:
updated to use `rm`